dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1558,10 +1558,6 @@ a.download-video {
   }
 }
 
-#angular-my-projects-wrapper .gallery {
```

Comment:
It looks like these styles could be removed from this file too:
* `#section-management`
* `#landingpage #edit_teachers`